### PR TITLE
client: Spanish Localization

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**
- Pull requests adds localization in Spanish to OTT.

### CODE REVIEW

**Comment by Victor-M-Giraldo** [comment]
Seeing lots of errors for the lint. Could it be the upside down question marks/exclamation points, or letters with accents on them?

File: client/src/components/navbar/LocaleSelector.vue [hunk 1]
```diff
@@ -30,6 +30,10 @@ const locales = [
 		text: "🇷🇺",
 		value: "ru",
 	},
+	{
+		text: "🇪🇸",
+		value: "sp",
```
Comment (dyc3) [inline client/src/components/navbar/LocaleSelector.vue:35]: Should be `es`

**Review by dyc3 — APPROVED** [review]
Looks good, just needs a minor fix

**Comment by Victor-M-Giraldo** [comment]
Ah forgot a few. I'll change those.

**Comment by Victor-M-Giraldo** [comment]
I did run yarn lint, it's giving me errors on my desktop though. I'll run it on my laptop and push again in a second.

**Review by dyc3 — APPROVED** [review]

**Pull request merged**